diff --git a/rtcdtlstransport.py b/rtcdtlstransport.py
index 566b9cc..2dc320f 100644
--- a/rtcdtlstransport.py
+++ b/rtcdtlstransport.py
@@ -152,7 +152,7 @@ class RTCCertificate:
         )
         ctx.use_certificate(self._cert)
         ctx.use_privatekey(self._key)
-        ctx.set_cipher_list(b"HIGH:!CAMELLIA:!aNULL")
+        ctx.set_cipher_list(b"ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES256-SHA:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-CHACHA20-POLY1305")
         ctx.set_tlsext_use_srtp(b"SRTP_AES128_CM_SHA1_80")

         return ctx
